libvirt/tests/src/sriov/locked_memory_check/sriov_with_hotplug_memory.py

```diff
@@ -105,7 +105,7 @@ def hotplug_mem_device(vm_name, mem_dict, current_mem):
         test.log.info("TEST_STEP: Start the vm and check locked memory limit.")
         vm.start()
         test.log.debug("The current guest xml is:%s", virsh.dumpxml(vm_name).stdout_text)
-        vm_session = vm.wait_for_serial_login(timeout=240).close
+        vm_session = vm.wait_for_login(timeout=240).close
```

Comment:
May I ask reason of this change?

Reply:
The guest can't login with vm.wait_for_serial_login() but can login with wait_for_login() function. I'm not sure the reason but I think maybe this configuration check can't be got from serial console.

Reply:
How do you use vm_session variable? If you just want to login vm to check the accessibility, please close it directly. 
```suggestion
        vm.wait_for_login(timeout=240).close()
```

Reply:
A low-level mistake. Updated it now.